### DISCUSSION TITLE
New settings are terrible : (

### DISCUSSION THREAD

**Discussion by HuemanInstrument** [Mikubill/sd-webui-controlnet#1151] [General]
using deforum, trying to do a motion video with the exact settings I had for this one https://www.youtube.com/watch?v=gkOZfnEbZa4

And, this whole "Balanced
My prompt is more important
ControlNet is more important" is probably the change that caused these new settings to be terrible
I cannot find a setting that will allow me to retain the facial features but have everything else zooming, it's really disapointing.

"my prompt is more important" is the only setting that even looks mildly ok, and it's not retaining any of the facial features when I zoom in.  : ( 

It was really sad because I rendered this whole video in anticipation and then I updated deforum / automatic1111 to hopefully fix some other issue I was having, and now I really can't even do this project anymore.

**Comment by lllyasviel** [comment]
" balanced" uses exactly same codes as previous 1.0
you may compare the " balanced" results to the 1.0 results ( see also the readme for a section of using previous version) and you will find them same, pixel by pixel, if you use same random seed.
if you get results different from your expectations, it is most likely because of inaccurate human feeling and inaccurate human memory of past things, or bug of deform, or ( very unlikely) bug of ControlNet (in this case please open an issue)
